### PR TITLE
Updating Avalonia for .Net 9 (Browser) Compatibility

### PR DESCRIPTION
**Pull request by inforithmics**
Avalonia is updated so that it works with .Net 9 Targets and Browser. Is taken from this pull request. https://github.com/Mapsui/Mapsui/pull/2744

### CODE REVIEW

File: .github/workflows/dotnet-release-nugets.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
         echo $VERSION_OF_RELEASE
 
     - name: install workloads
-      run: dotnet workload install maui macos android ios maccatalyst wasm-tools wasm-tools-net8
```
Comment (pauldendulk) [inline .github/workflows/dotnet-release-nugets.yml:45]: The wasm-tools-net8 tools where only for Avalonia?

File: .github/workflows/dotnet-docs.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ jobs:
       uses: actions/setup-dotnet@v4
       with:
          dotnet-version: |
-           9.0.x
```
Comment (pauldendulk) [inline .github/workflows/dotnet-docs.yml:45]: Why back to 8 in the docs? This seems not related to Avalonia? Is it by mistake? Or does this revert an earlier error?

Comment (inforithmics) [reply .github/workflows/dotnet-docs.yml:45]: No this is an error, will revert that

File: .github/workflows/dotnet-release-nugets.yml (hunk 1 above)
Comment (inforithmics) [reply .github/workflows/dotnet-release-nugets.yml:45]: No this is an error, will revert that

File: .github/workflows/dotnet.yml [hunk 3]
```diff
@@ -218,8 +218,8 @@ jobs:
       uses: NuGet/setup-nuget@v2
       with:
         nuget-version: latest
-    - name: install maui macos android ios maccatalyst wasm-tools wasm-tools-net8 wasm-tools wasm-tools-net8
-      run: dotnet workload install maui macos android ios maccatalyst wasm-tools wasm-tools-net8 wasm-tools wasm-tools-net8
+    - name: install maui macos android ios maccatalyst wasm-tools wasm-tools-net8
```
Comment (inforithmics) [inline .github/workflows/dotnet.yml:221]: wasm-tools and wasm-tools-net8 was duplicated

**Review by pauldendulk — APPROVED** [review]
👍
